extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -98,7 +98,12 @@ public static function onWikiaSkinTopScripts( &$vars, &$scripts ) {
 		$vars['ads'] = [
 			'context' => $adContext,
 			'runtime' => [
-				'sp' => []
+				'pf' => [
+					'blocking' => null
+				],
+				'sp' => [
+					'blocking' => null
```

Comment:
Minor one!
I believe `null` values is more error prone and in order to check detection in console I should type more chars :wink: